package.json

```diff
@@ -48,7 +48,8 @@
     "@typescript-eslint/eslint-plugin": "^5.42.0",
     "@typescript-eslint/parser": "^5.42.0",
     "@uniswap/sdk-core": "^3.1.0",
-    "@walletconnect/ethereum-provider": "^1.8.0",
+    "@walletconnect/ethereum-provider-v1": "npm:@walletconnect/ethereum-provider@^1.8.0",
+    "@walletconnect/ethereum-provider-v2": "npm:@walletconnect/ethereum-provider@^2.4.6",
```

Comment:
I wonder if Yarn would automatically deduplicate this with a regular `2.x` `@walletconnect/ethereum-provider` that interface app is already running to avoid duplicates. Looking at the `yarn.lock`, I see they're represented by a different name, so I am not sure.

> walletconnect/ethereum-provider-v1@npm:@walletconnect/ethereum-provider@^1.8.0":

Reply:
These are devDependencies, as they're only used for types, so it won't be hoisted at all :)